### PR TITLE
TST add test case for drop_duplicates

### LINKED ISSUE #35179: TST verify inplace ops return None
test if pandas methods, when called inplace, return None

drop_duplicates https://github.com/pandas-dev/pandas/pull/35121. **merged**
dropna https://github.com/pandas-dev/pandas/pull/35180

### PR DESCRIPTION
**Pull request by r-toroxel**
Add a test case to drop_duplicates for inplace=True.

### CODE REVIEW

**Review by mroeschke — COMMENTED** [review]
Does this close a particular issue?

**Comment by r-toroxel** [comment]
No.
Its just expanding on test coverage, after looking at pytest-cov and existing tests.

File: pandas/tests/frame/methods/test_drop_duplicates.py [hunk 1]
```diff
@@ -425,3 +425,9 @@ def test_drop_duplicates_null_in_object_column(nulls_fixture):
     df = DataFrame([[1, nulls_fixture], [2, "a"]], dtype=object)
     result = df.drop_duplicates()
     tm.assert_frame_equal(result, df)
+
+
+def test_drop_duplicates_inplace_result():
+    df = DataFrame({"a": [1, 2, 3], "b": [1, 2, 4], "c": [1, 6, 5]})
+    result = df.drop_duplicates(inplace=True)
+    assert result is None
```
Comment (jreback) [inline pandas/tests/frame/methods/test_drop_duplicates.py:433]: can you use assert_frame_equal(df, expected) and construct the expected.

ideally this test goes with other basic drop_duplicates call and we parameterize on inplace

**Review by jreback — CHANGES_REQUESTED** [review]

File: pandas/tests/frame/methods/test_drop_duplicates.py (hunk 1 above)
Comment (r-toroxel) [reply pandas/tests/frame/methods/test_drop_duplicates.py]: yes. just checking the return value goes well with the existing 
`test_drop_duplicates_inplace`
moved it there.

File: pandas/tests/frame/methods/test_drop_duplicates.py [hunk 2]
```diff
@@ -333,64 +333,73 @@ def test_drop_duplicates_inplace():
     )
     # single column
     df = orig.copy()
-    df.drop_duplicates("A", inplace=True)
+    return_value = df.drop_duplicates("A", inplace=True)
     expected = orig[:2]
     result = df
     tm.assert_frame_equal(result, expected)
+    assert(return_value is None)
```
Comment (mroeschke) [inline pandas/tests/frame/methods/test_drop_duplicates.py:340]: Nit: You can remove the parenthesis from all of these `assert`s

Comment (r-toroxel) [reply pandas/tests/frame/methods/test_drop_duplicates.py]: thx!

**Review by mroeschke — APPROVED** [review]

**Review by jreback — APPROVED** [review]

**Comment by jreback** [comment]
thanks @r-toroxel 

happy to take PRs to verify other inplace ops return None

**Comment by r-toroxel** [comment]
thanks. yes. created an issue. https://github.com/pandas-dev/pandas/issues/35179

**Pull request merged**